speech.yaml

```diff
@@ -285,6 +285,17 @@ components:
             in key/value format, where the key will be replaced
             by the value in transcription.
           default: {}
+        oov:
+          type: object
+          additionalProperties:
+            type: string
+          description:
+            A set of custom vocabulary terms with optional soundouts. 
+            Accepts JSON dictionary and list formats, or a comma-
+            delineated string of text. Custom vocabulary improves
+            transcription for accuracy words that are not in the 
+            language model's standard vocabulary.
+          default: {}
```

Comment:
Consider adding an `example` section, especially since it looks like this supports multiple formats.

https://github.com/medallia/public-api-swaggers/blob/eb0159d96de5d7b55a60af04e6c67af32e747302/speech.yaml#L362